src/ia-book-actions.js

```diff
@@ -347,17 +347,158 @@ export default class IABookActions extends LitElement {
   }
 
   /**
+   * create / select the modal-manager component on DOM
+   * @memberof IABookActions
+   */
+  async useModalManager() {
+    if (!this.modal) {
+      this.modal = document.querySelector('modal-manager');
+
+      if (!this.modal) this.modal = document.createElement('modal-manager');
+    }
+
+    this.modal.id = 'action-bar-modal';
+    await document.body.appendChild(this.modal);
+  }
+
+  /**
+   * Show the waring modal to ask user if they are still reading
+   */
+  async showWarningModal() {
+    await this.useModalManager();
+
+    // if secondsLeft < 60, consider it 1 minute
+    let { secondsLeft } = this.loanRenewResult;
+    if (secondsLeft === undefined) {
+      secondsLeft = this.lendingStatus.secondsLeftOnLoan;
+    } else {
+      secondsLeft = secondsLeft > 60 ? secondsLeft : 60;
+    }
+
+    const config = new ModalConfig();
+    config.headline = 'Are you still here?';
+    config.showCloseButton = false;
+    config.closeOnBackdropClick = false;
+    config.message = this.loanRenewHelper?.getMessageTexts(
+      this.loanRenewResult.texts,
+      secondsLeft
+    );
+
+    const customContent = html`<br />
```

Comment:
`<br>` is sufficient since we aren't aiming for XHTML compatibility. HTML5 doesn't require the closing slash on it.